### PR TITLE
Add YANG model and unit tests for additional structured events

### PR DESCRIPTION
**Pull request by zbud-msft**
#### Why I did it

Added YANG models for additional events for host, swss, and dhcp relay

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

### CODE REVIEW

File: src/sonic-yang-models/yang-models/sonic-events-host.yang [hunk 1]
```diff
@@ -179,5 +179,113 @@ module sonic-events-host {
...
+                    }
+                    length 1..32 {
+                        error-message "Invalid length for process not running process_name.";
+                        error-app-tag process-not-running-invalid-process-name-length;
+                    }
+                }
+                description "Name of process that is not running";
+            }
+
+            leaf ctr_name {
```
Comment (ganglyu) [inline src/sonic-yang-models/yang-models/sonic-events-host.yang:274]: ctr_name and process_name are used repeatedly, maybe we can add them to sonic-type.yang?

File: src/sonic-yang-models/yang-models/sonic-events-swss.yang [hunk 2]
```diff
@@ -104,5 +104,41 @@ module sonic-events-swss {
...
+                    }
+                    length 1..32 {
+                        error-message "Invalid length for select operation failure operation_result.";
+                        error-app-tag select-operation-failure-invalid-operation-result-length;
+                    }
+                }
+                description "Select operation result";
+            }
+
+            leaf command {
```
Comment (ganglyu) [inline src/sonic-yang-models/yang-models/sonic-events-swss.yang:128]: description?

**Comment by ganglyu** [comment]
Please update sample_config_db.json

File: src/sonic-yang-models/yang-models/sonic-events-host.yang (hunk 1 above)
Comment (zbud-msft) [reply src/sonic-yang-models/yang-models/sonic-events-host.yang:241]: Done

File: src/sonic-yang-models/yang-models/sonic-events-swss.yang (hunk 2 above)
Comment (zbud-msft) [reply src/sonic-yang-models/yang-models/sonic-events-swss.yang:128]: Added

**Comment by zbud-msft** [comment]
> Please update sample_config_db.json

Structured events will not be stored in config db

**Review by ganglyu — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**